src/services/ghost/__tests__/GhostModel.spec.ts

```diff
@@ -348,7 +348,7 @@ describe("GhostModel", () => {
 			})
 
 			it("stores and returns profile name after loading", async () => {
-				const supportedProviders = Object.keys(AUTOCOMPLETE_PROVIDER_MODELS)
+				const supportedProviders = [...AUTOCOMPLETE_PROVIDER_MODELS.keys()]
```

Comment:
noise?  or signal?

(fine either way, just curious)

Reply:
Opus said

[src/services/ghost/__tests__/GhostModel.spec.ts](vscode-webview://1n3era6ha6ld3ktpthg0f26upj33ab2kh1oducguj98qv9gqc1t8/src/services/ghost/__tests__/GhostModel.spec.ts:351-410) - Fixed three tests that incorrectly used Object.keys(AUTOCOMPLETE_PROVIDER_MODELS) on a Map (which returns an empty array). Changed to [...AUTOCOMPLETE_PROVIDER_MODELS.keys()] to properly get the provider keys.